### ISSUE TITLE
Set a default way to install the whole repo?

### ISSUE THREAD

**Issue by apayne97** [choderalab/asapdiscovery#588]
pls send help

```
(asapdiscovery) paynea@lilac-ln02:/data/chodera/paynea/asapdiscovery$ pip install -e asapdiscovery-*
Obtaining file:///lila/data/chodera/paynea/asapdiscovery/asapdiscovery-alchemy
  Installing build dependencies ... done
  Checking if build backend supports build_editable ... done
  Getting requirements to build editable ... done
  Installing backend dependencies ... done
  Preparing editable metadata (pyproject.toml) ... done
ERROR: Could not find a version that satisfies the requirement asapdiscovery-cli (from versions: none)
ERROR: No matching distribution found for asapdiscovery-cli
```

EDIT: asap-alchemy depends on asapdiscovery-cli, so asapdiscovery-cli needs to be installed first

**Comment by apayne97** [comment]
the data, modeling, and docking subpackages all seem to work fine

**Comment by apayne97** [comment]
Oh, it's just that `asapdiscovery-cli` needs to be installed first

**Comment by apayne97** [comment]
Probably the solution is #20

**Comment by hmacdope** [comment]
I think we can just remove explicit dep in `pyproject.toml` for `asap-alchemy` and then just have the import fail @jthorton ?

On a broader note there is a way to make a namespace package installable from the root pyproject.toml, but it is fiddly. I will investigate.

**Comment by hmacdope** [comment]
Fixed in #731 (at least semi)

**Comment by hmacdope** [comment]
Closing
